src/MusicAnnouncementsMod.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using BepInEx;
 using Music;
+using System.Diagnostics;
 using System.Security.Permissions;
```

Comment:
The `Debug` class in `Debug.Log()` is from `UnityEngine` rather than `System.Diagnostics` (and having both included here generates an error), so this can be removed:
```suggestion
using Music;
using System.Security.Permissions;
```